m4_sessions/1708-inning/group_one.md

```diff
@@ -1,9 +1,32 @@
 # Group One
 
 ### Session Title:
+* Transhumanism - 
 
 ### Session Outcomes: 
+* Create an understanding of what transhumanism is.
+* How does this relate to us as developers of software?
+* History, now, and future of transhumanism
+* Discussion humanism and see how people's feelings and thoughts change after the session.
 
 ### Session Outline:
+* Whiteboard spectrum of how people feel about this.
+  - Have everyone go one by one, introduce themself and point out where they are in the spectrum.
+* Introduction to material
+  - History, current
+* small group discussions
+  - After we have groups discuss a main point or two, we have another pulse check on what people thing of transhumanism.
+* Future of transhumanism
+  - What role do we as developers play
+* another small group discussion
+  - last pulse check and see where people are in how they feel about this
```

Comment:
Have another set of questions that pushes boundaries or also serves as devil's advocate in case everyone is agreeing.